front/src/routes/integration/all/mqtt/setup-page/actions.js

```diff
@@ -0,0 +1,67 @@
+import createActionsIntegration from '../../../../../actions/integration';
+import { RequestStatus } from '../../../../../utils/consts';
+
+const createActions = store => {
+  const integrationActions = createActionsIntegration(store);
+  const actions = {
+    async loadProps(state) {
+      let mqttURL;
+      let mqttUsername;
+      let mqttPassword;
+      try {
+        mqttURL = await state.httpClient.get('/api/v1/service/mqtt/variable/MQTT_URL');
+        mqttUsername = await state.httpClient.get('/api/v1/service/mqtt/variable/MQTT_USERNAME');
+        if (mqttUsername.value) {
+          mqttPassword = 'FAKE_PASSWORD';
```

Comment:
Why do you put a fake password?

Reply:
To not retrieve the real one in a "clear" string, and to get a ***** string in password field to tell the user that a password is set.